README.rst

```diff
@@ -19,33 +13,39 @@ the project so far.**
 
 This Napp allows a user to create a point to point L2 Ethernet Virtual Circuit.
 
-When fully implemented, this napp will provide a REST API to create/modify/delete circuits. For now, the list of installed circuits is kept in memory,
-but it should be later kept in a permanent storage.
-Circuits will be installed at the time of request or at a predefined time, and can also have a time to be deleted.
-The Napp also will listen for PortStatus events, modifying circuits that use a port that went down.
+Features
```

Comment:
@ajoaoff @italovalcy should we also remove this paragraph here, seems it was related to the experimental stuff that we're removing

>     THIS NAPP IS STILL EXPERIMENTAL AND IT'S EVENTS, METHODS AND STRUCTURES MAY CHANGE A LOT ON THE NEXT FEW DAYS/WEEKS, USE IT AT YOUR OWN DISCERNMENT

Reply:
I think so, as we changed the stability to stable.

Reply:
Done